compiler/packages/babel-plugin-react-compiler/src/HIR/Globals.ts

```diff
@@ -362,6 +362,17 @@ const REACT_APIS: Array<[string, BuiltInType]> = [
       returnValueKind: ValueKind.Mutable,
     }),
   ],
+  [
```

Comment:
This is left over from a previous version of this work, but seems reasonable to add anyways